pkg/vsphere/spec/spec_test.go

```diff
@@ -29,7 +29,7 @@ import (
 func URL(t *testing.T) string {
```

Comment:
@dougm unfortunately that would cause a dependency cycle between the `test` and `spec` packages. I suspect this is why @caglar10ur redefined the `URL` method in `spec_test.go`. Is it worth extracting it out into a `testutils` package?